bin/i18n/i18n_script_utils.rb

```diff
@@ -437,12 +396,14 @@ def self.rename_dir(from_dir, to_dir)
     FileUtils.rm_r(from_dir)
   end
 
-  def self.delete_empty_crowdin_locale_dir(crowdin_locale)
```

Comment:
I did a search across the repo and at first look, the method doesn't seem to be used anywhere else. 👍